packages/slate-editor/src/modules/popper-options-context/types.ts

```diff
@@ -1,9 +1,11 @@
 import type { PreventOverflowModifier } from '@popperjs/core/lib/modifiers/preventOverflow';
+import type { RefObject } from 'react';
 
 export interface Context {
     modifiers?: {
         preventOverflow?: PreventOverflowModifier['options'];
     };
     placement?: 'left-start' | 'right-start';
+    autoPlacementPortal?: RefObject<HTMLElement | undefined>;
```

Comment:
Would it be better to split this into two props?

I could imagine someone wanting just to use the portal and someone using both the auto placement and portal.

Maybe you could add `auto` as a possible option to `placement`? Then, if `placement === auto`, you can use `allowedAutoPlacements` property to specify `right-start` and `left-start`.

Reply:
@aspirisen I see you've asked for my review again but I've left a comment here yesterday. Can you check it out?

Reply:
Sorry, forgot to push :)